presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -238,6 +238,7 @@
     public static final String LEAF_NODE_LIMIT_ENABLED = "leaf_node_limit_enabled";
     public static final String PUSH_REMOTE_EXCHANGE_THROUGH_GROUP_ID = "push_remote_exchange_through_group_id";
     public static final String OPTIMIZE_MULTIPLE_APPROX_PERCENTILE_ON_SAME_FIELD = "optimize_multiple_approx_percentile_on_same_field";
+    public static final String REWRITE_IF_ON_AGGREGATION_TO_AGGREGATION_IF = "rewrite_if_on_aggregation_to_aggregation_if";
```

Comment:
Let's call it optimize_conditional_aggregations or something easier to understand

Reply:
Done

Reply:
I would like to use optimize as the option, not rewrite because we optimize plans and rewrite queries :)